src/web/src/components/Footer.vue

```diff
@@ -7,10 +7,14 @@
           <b-col>
             <!-- TODO: Autogenerate these when doing the user side, semester select -->
             <strong class="section-head">Other Semesters</strong>
-            <a class="link" href=#>Fall 2019</a>
-            <a class="link" href=#>Fall 2020</a>
-            <a class="link" href=#>Spring 2020</a>
-            <a class="link" href=#>Summer 2020</a>
+            <a 
+              v-for="semester in semesterOptions"
+              v-bind:key="semester.text"
+              v-bind:value="semester.value"
```

Comment:
1) `:` is shorthand for `v-bind:`
2) does `<a>` need a `:value` prop?

Reply:
There is an error produced if :value is not given :(